indico/modules/events/papers/templates/display/_paper_list.html

```diff
@@ -0,0 +1,66 @@
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+
+{% macro render_user_papers_list(event, contributions, to_review=false, reviewed=false) %}
+    <section>
+        {% if to_review %}
+            {% set reviewing_list_id = 'to-review-list' %}
+        {% elif reviewed %}
+            {% set reviewing_list_id = 'reviewed-list' %}
+        {% else %}
+            {% set reviewing_list_id = '' %}
+        {% endif %}
+        <div id="{{ reviewing_list_id }}" class="display-contribution-list">
```

Comment:
Do you ever get into the else case where you set no ID? If yes, then add another if here to avoid `id=""` which is most likely invalid